specification/keyvault/data-plane/Security.KeyVault.Secrets/routes.tsp

```diff
@@ -0,0 +1,289 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace KeyVault;
```

Comment:
Either here or in a client TSP, this or something else should be changed so we generate a `SecretClient` and not a `KeyVaultClient`.